src/css/support.js

```diff
@@ -53,6 +53,11 @@ define( [
 		div = null;
 	}
 
+	// Rounding up the exact checking match values
```

Comment:
I find this comment confusing. And given the clarity of the function body, it can probably just be dropped.

Reply:
I dropped the comment as suggested by you.